expected/15.4/extra/aggregates.out

```diff
@@ -86,6 +86,46 @@ SELECT avg(a) AS avg_32 FROM aggtest WHERE a < 100;
  32.6666666666667
 (1 row)
 
+--Testcase 2+1:
```

Comment:
Could you please follow the current style TestcaseID? New TestcaseID = max of current TestcaseID + 1

Reply:
Rewritten to chaotic numbering style - 697..708 after maximal existed TestcaseID. Please verify.

Reply:
Thank you for supporting. It's OK